src/server/saved_objects/service/lib/repository.js

```diff
@@ -400,10 +436,15 @@ export class SavedObjectsRepository {
     const response = await this._callCluster('mget', {
       index: this._index,
       body: {
-        docs: objects.map(object => ({
-          _id: this._serializer.generateRawId(namespace, object.type, object.id),
-          _type: this._type,
-        }))
+        docs: objects.map(object => {
+          if(!this._isTypeAllowed(object.type)) {
```

Comment:
Similar to `bulkCreate` above, `bulkGet` should have a response for each requested object, so we probably shouldn't filter out unsupported types here.

Reply:
@legrego Please take a look at the updates, I'm not sure about expectation of ordering or anything with these requests!?

Reply:
I do think you'll want to maintain order. If caller passes in [A, B, C], we shouldn't be returning [B, C, A]